src/app/my-journeys-listing/my-journeys-listing.page.ts

```diff
@@ -0,0 +1,92 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { ProfileService } from '../services/profile/profile.service';
+import { LoaderService } from '../services/loader/loader.service';
+import { ProjectsApiService } from '../services/projects-api/projects-api.service';
+import { ToastService } from '../services/toast/toast.service';
+import urlConfig from 'src/app/config/url.config.json';
+
+@Component({
+  selector: 'app-my-journeys-listing',
+  templateUrl: './my-journeys-listing.page.html',
+  styleUrls: ['./my-journeys-listing.page.scss'],
+})
+export class MyJourneysListingPage  {
+  headerConfig = {
+    title:"MY_JOURNEYS"
+  }
+  myjourneys :any[]=[];
+  page = 1;
+  limit = 15;
+  count = 0;
+  disableLoading: boolean = false;
+  pageConfig: any = {};
+  profilePayload: any = {};
+
+  constructor(
+    private route:Router,
+    private profileService: ProfileService,
+    private loaderService: LoaderService,
+    private projectsApiService: ProjectsApiService,
+    private toastService: ToastService
+  ) { }
+
+  ionViewWillEnter(){
+    this.getProfileDetails();
```

Comment:
Do we need profile details for this page too?

Reply:
no removed